proto/pdpb.proto

```diff
@@ -588,6 +588,9 @@ message StoreStats {
 message PeerReport {
     raft_serverpb.RaftLocalState raft_state = 1;
     raft_serverpb.RegionLocalState region_state = 2;
+    bool is_force_leader = 3;
```

Comment:
Typo? is_forced_leader?

Reply:
no, I'd like use `force_leader` to keep consistency with TiKV